test/lit/passes/gto-removals.wast

```diff
@@ -850,7 +850,7 @@
   (func $unreachable-set-2 (param $"{mut:i8}" (ref null $"{mut:i8}"))
     ;; As above, but the side effects now are a br. Again, the br must happen
     ;; before the trap (in fact, the br will skip the trap here).
-    (block
+    (block $block
```

Comment:
Weird, how did this parse and validate before..?

Reply:
Because the parser magically assigned the default name "$block" and it happened to match 😆